docker/docker-compose.yml

```diff
@@ -1,3 +1,4 @@
+version: "3"
```

Comment:
Is `version` necessary here? (Just asking because I don't know that)

Reply:
I got error on Ubuntu when I was trying to run it without version. On Windows with docker desktop it works correctly without version.